test/public/overview.test.js

```diff
@@ -17,32 +17,27 @@
  */
 
 const assert = require('assert');
-const puppeteer = require('puppeteer');
 const pti = require('puppeteer-to-istanbul');
-const { server } = require('../../lib/application');
+const puppeteer = require('puppeteer');
 
 module.exports = function () {
     // Configure this suite to have a default timeout of 10s
-    this.timeout(10000);
+    this.timeout(5000);
 
     let page;
     let browser;
-    let PORT;
+    const URL = 'http://localhost:4000';
```

Comment:
port number (and address) are configurable thus having them hard coded is not good.